src/service-disruptions/ServiceDisruptionSheet.tsx

```diff
@@ -0,0 +1,60 @@
+import {BottomSheetContainer} from '@atb/components/bottom-sheet';
+import Button from '@atb/components/button';
+import FullScreenFooter from '@atb/components/screen-footer/full-footer';
+import {ScreenHeaderWithoutNavigation} from '@atb/components/screen-header';
+import * as Sections from '@atb/components/sections';
+import {
+  ServiceDisruptionsTexts,
+  ScreenHeaderTexts,
+  useTranslation,
+} from '@atb/translations';
+import React, {forwardRef} from 'react';
+import {Linking, View} from 'react-native';
+import {ScrollView} from 'react-native-gesture-handler';
+import ThemeText from '../components/text';
+
+type Props = {
+  close: () => void;
+  serviceDisruptionUrl: string;
+};
+
+const ServiceDisruptionSheet = forwardRef<View, Props>(
+  ({close, serviceDisruptionUrl}, focusRef) => {
+    const {t} = useTranslation();
+
+    return (
+      <BottomSheetContainer>
+        <ScreenHeaderWithoutNavigation
+          title={t(ServiceDisruptionsTexts.header.title)}
+          leftButton={{
+            type: 'cancel',
+            onPress: close,
+            text: t(ScreenHeaderTexts.headerButton.cancel.text),
+            testID: 'cancelButton',
+          }}
+          color={'background_2'}
+          setFocusOnLoad={false}
+        />
+
+        <Sections.Section withFullPadding>
+          <View ref={focusRef}>
```

Comment:
Jeg så Espen meldte om en feil på dette. Det demrer noe i hodet mitt om at det på Android må være gitt `accessible={true}` på View` for at fokus skal fungere.